### DISCUSSION TITLE
any ADVICE on transforming my microbiota dataset before ordination?

### DISCUSSION THREAD

**Discussion by marwa38** [vegandevs/vegan#549] [Q&A]

**Comment by gavinsimpson** [comment]
You are likely to get better informed answers by asking the developer(s) of {phyloseq}

**Comment by jarioksa** [comment]
I have no idea what are **phyloseq** objects. Better ask **phyloseq** developers.

**Comment by eduardszoecs** [comment]
Some guidance with focus on microbio can be found here: https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.13536

I don't think that there is a one fits all method. It depends on your data, questions, methods...

**Comment by jarioksa** [comment]
These questions really are specific to the field of science and its internal traditions and rules. You better ask microbio/phyloseq communities about their recommendations. They probably differ from the ones I'd give. This also concerns Discussion #495.
